### PR TITLE
Feature/permission endpoints

### PR DESCRIPTION
**Pull request by marioncottard**

### CODE REVIEW

**Comment by marioncottard** [comment]
Linked to https://github.com/baobabsoluciones/cornflow-server/issues/133

File: cornflow/endpoints/permission.py [hunk 1]
```diff
@@ -40,8 +46,29 @@ def get(self):
...
+    @Auth.auth_required
+    @use_kwargs(PermissionViewRoleRequest, location="json")
+    @marshal_with(PermissionViewRoleResponse)
+    def post(self, **kwargs):
+        AUTH_TYPE = current_app.config["AUTH_TYPE"]
+        if AUTH_TYPE == AUTH_LDAP:
+            raise EndpointNotImplemented(
+                "The permissions have to be created in the directory."
+            )
+
```
Comment (ggsdc) [inline cornflow/endpoints/permission.py:59]: Actually, if I remember this correctly, but I should review it, the permissions are not defined in the directory, just the roles, but the permission assigned to each role should be done in cornflow. Do not fix it just yet until I review it.

**Review by ggsdc — APPROVED** [review]
I'm going to approve it, but I will not merge just yet, cause I have to review a couple of things

**Review by ggsdc — APPROVED** [review]
I've made a small change myself. Even using LDAP permissions are handled on our end

**Pull request merged**